examples/kafka/adobe-s3-connector/README.md

```diff
@@ -102,34 +102,37 @@ $ kanctl create profile s3compliant --access-key <aws-access-key> \
 $ kubectl create -f ./kafka-blueprint.yaml -n kanister
 ```
 
-## Perform Backup
-To perform backup to S3, an ActionSet is created to run `kafka-connect`.
-```bash
-# Create an actionset
-$ kanctl create actionset --action backup --namespace kanister --blueprint kafka-blueprint --profile kafka-test/s3-profile-fn64h --objects v1/configmaps/kafka-test/s3config
-```
-## Verify the backup
-We can verify the backup operation by adding some data to the topic configured earlier
+## Insert Data in Topic
+* Create a topic `blogs` on Kafka server. The `blogs` topic is configured as source and sink topics in `s3config` configmap
 
-* List all topics in Kafka server
-```bash
-$ kubectl -n kafka-test run kafka-producer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-topics.sh --bootstrap-server=my-cluster-kafka-bootstrap:9092 --list
-```
-* Create a topic on Kafka server
 ```bash
 $ kubectl -n kafka-test run kafka-producer -ti --image=strimzi/kafka:0.20.0-kafka-2.6.0 --rm=true --restart=Never -- bin/kafka-topics.sh --create --topic blogpost --bootstrap-server my-cluster-kafka-bootstrap:9092
 ```
-* Create a producer to push data to blogpost topic
+* Create a producer to push data to `blogs` topic
```

Comment:
is there any specific reason to change blogpost to blogs? If not, let's keep `blogposts`.

Reply:
Yeah, we need to change it, because in the configuration files we are using `blogs` as topics.